### PR TITLE
Cursor added for button and improved some css

### PR DESCRIPTION
**Pull request by nileshlad09**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (immattdavison) [inline .vscode/settings.json:1]: Please do not commit your .vscode folder - this can be added to the .gitignore

**Review by immattdavison — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (nileshlad09) [reply .vscode/settings.json:1]: hi can you please help me how i can change it now

File: popup.html [hunk 2]
```diff
@@ -15,13 +15,16 @@
...
+            </div>
         </div>
-        <div class="power-switch">
-            <button onclick="onOff()">Toggle NMD</button>
-        </div>
+        
+    </div>
     </body>
 
 </html>
```
Comment (immattdavison) [inline popup.html:30]: The button should not be within the `.title-block` div.

**Review by immattdavison — CHANGES_REQUESTED** [review]

File: popup.html (hunk 2 above)
Comment (nileshlad09) [reply popup.html:30]: No it is not in  .title-block 
It is in .container

Comment (nileshlad09) [reply popup.html:30]: Any other changes ?

Comment (immattdavison) [reply popup.html:30]: image

**Comment by nileshlad09** [comment]
193409169-e3a6f71d-ba4b-4bed-b98c-c98a744e5393

**Pull request closed without merging**